purchase_request/views/purchase_request_view.xml

```diff
@@ -249,6 +252,7 @@
                     <field name="analytic_account_id"
                            groups="analytic.group_analytic_accounting"/>
                     <field name="supplier_id"/>
+                    <field name="cancelled"/>
```

Comment:
I'm not sure that the user would realize about why the line is gray, that's why I prefer to show the field but yes, maybe you are right and I have to trust more the user :sweat_smile: